update.sh

```diff
@@ -4,10 +4,8 @@ REPO="anyproto/anytype-heart"
 FILE="addon.tar.gz"
 GITHUB="api.github.com"
 
-user=$1
-token=$2;
-platform=$3;
-arch=$4;
+platform=${1:-ubuntu-latest};
```

Comment:
is it a default value? should not be ${1:=ubuntu-latest} ?

Reply:
Well.. we could use that form, but it'd then be more like:

```
platform=$1
: ${platform:=ubuntu-latest}
```

Ref: https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html

Reply:
oh, ok sorry, ima noob in bash

Reply:
No worries! I got curious and had to look up `:=` myself :)